tests/unit_tests/test_wave_06.py

```diff
@@ -76,16 +76,23 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That the results is truthy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert result
+    assert len(jesse.inventory) == 3
+    assert len(tai.inventory) == 3
+    assert item_f in tai.inventory
+    assert item_c in jesse.inventory
+    assert item_c not in tai.inventory
+    assert item_f not in jesse.inventory
```

Comment:
Great asserts! It would be a good idea to check that the other items in their inventories are still there as well.